mewarpx/examples/test_install_fieldsolver.py

```diff
@@ -0,0 +1,277 @@
+"""
+Monte-Carlo Collision script benchmark against case 1 results from
+Turner et al. (2013) - https://doi.org/10.1063/1.4775084.
+Shortened for quick runs.
+"""
+import numpy as np
+import time
+
+from pywarpx import picmi
+
+# init_libwarpx must be run before importing some mewarpx modules
+from mewarpx import util as mwxutil
+mwxutil.init_libwarpx(ndim=2, rz=False)
+from pywarpx import _libwarpx
+
+# from pywarpx import fields
+from pywarpx import callbacks
+
+from mewarpx.mwxrun import mwxrun
+from mewarpx.diags_store import diag_base
+from mewarpx.poisson_psuedo_1d import PoissonSolverPseudo1D
+
+from minerva import util as minutil
+
+constants = picmi.constants
+
+##########################
+# physics parameters
+##########################
+
+D_CA = 0.067 # m
+
+N_INERT = 9.64e20 # m^-3
+T_INERT = 300.0 # K
+
+FREQ = 13.56e6 # MHz
+
+VOLTAGE = 450.0
+
+M_ION = 6.67e-27 # kg
+
+PLASMA_DENSITY = 2.56e14 # m^-3
+T_ELEC = 30000.0 # K
+
+##########################
+# numerics parameters
+##########################
+
+# --- Grid
+nx = 128
+ny = 16
+
+xmin = 0.0
+ymin = 0.0
+xmax = D_CA
+ymax = D_CA / nx * ny
+
+number_per_cell_each_dim = [32, 16]
+
+DT = 1.0 / (400 * FREQ)
+
+# Total simulation time in seconds
+TOTAL_TIME = 0.25 / FREQ # 1280 / FREQ
+# Time (in seconds) between diagnostic evaluations
+DIAG_INTERVAL = 0.05 / FREQ # 32 / FREQ
+
+# --- Number of time steps
+max_steps = int(TOTAL_TIME / DT)
+diag_steps = int(DIAG_INTERVAL / DT)
+diagnostic_intervals = "::%i" % diag_steps
+
+##########################
+# physics components
+##########################
+
+v_rms_elec = np.sqrt(minutil.kb_J * T_ELEC / minutil.m_e)
+v_rms_ion = np.sqrt(minutil.kb_J * T_INERT / M_ION)
+
+uniform_plasma_elec = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_elec] * 3,
+    directed_velocity = [0.] * 3
+)
+
+uniform_plasma_ion = picmi.UniformDistribution(
+    density = PLASMA_DENSITY,
+    upper_bound = [None] * 3,
+    rms_velocity = [v_rms_ion] * 3,
+    directed_velocity = [0.] * 3
+)
+
+electrons = picmi.Species(
+    particle_type='electron', name='electrons',
+    initial_distribution=uniform_plasma_elec
+)
+ions = picmi.Species(
+    particle_type='He', name='he_ions',
+    charge='q_e',
+    initial_distribution=uniform_plasma_ion
+)
+
+''' Add back when MCC initialization has been added to mewarpx
```

Comment:
For this sample that just tests installfieldsolver, eliminate this commented section - and the extraneous stuff at bottom of this script